### ISSUE TITLE
pressing 'm' on selected issue not working (help wanted)

### ISSUE THREAD

**Issue by gianpaolof** [ankitpokhrel/jira-cli#578]
hello, new windows user here. maybe I'm missing something but it seems I can't get the 'm' key working

**Describe the bug**
can't transition selected issue

**Please provide following details**
1. JiraCLI Version:
(Version="v1.3.0", GitCommit="ed84fe74f68ba1b2cf0504a931ca09a03b2c9fed", CommitDate="2023-01-02T09:59:32+00:00", GoVersion="go1.19.4", Compiler="gc", Platform="linux/amd64")
2. Are you using Jira cloud or on-premise jira server?
Version:         8.2.3
Build Number:    802003
Deployment Type: Server
Default Locale:
3. What operating system are you using? Also mention version.
windows wsl
4. What terminal are you using? Also mention version.
wsl subsystem

**To Reproduce**

list current sprint issue
press m 
nothing happens

**Comment by ankitpokhrel** [comment]
Hi @gianpaolof, this feature is not yet release. If you wish to use the feature you can either [compile the binary manually](https://github.com/ankitpokhrel/jira-cli#development) or use one of the following means to install dev build.

```sh
# Docker
docker run -it --rm ghcr.io/ankitpokhrel/jira-cli:main

# Golang
go install github.com/ankitpokhrel/jira-cli/cmd/jira@main

# Homebrew
$ brew install jira-cli --HEAD
```

**Comment by sf8193** [comment]
any chance you have a timeline for this release?

**Comment by ankitpokhrel** [comment]
@sf8193 that is already available in past releases? Is that not working for you?

**Comment by ankitpokhrel** [comment]
This is available in the new release [v1.4.0](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.4.0)
